08_model-compression.ipynb

```diff
@@ -634,6 +634,8 @@
     "        self.teacher_model = teacher_model\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
</pre><p>Since we already define a <code>device</code> object earlier in the notebook, I think we can skip it here (assuming readers run the code from start to finish :)) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>